lib/associations/base.js

```diff
@@ -1,3 +1,21 @@
-var Association = function () {};
+'use strict';
 
-module.exports = Association;
\ No newline at end of file
+var Association = function() {};
+
+// Normalize input - may be array or single obj, instance or primary key - convert it to an array of built objects
+Association.prototype.toInstanceArray = function (objs) {
+  if (!Array.isArray(objs)) {
+    objs = [objs];
+  }
+  return objs.map(function(obj) {
+    if (!(obj instanceof this.target.Instance)) {
+      var tmpInstance = {};
+      tmpInstance[this.target.primaryKeyAttribute] = obj;
+      return this.target.build(tmpInstance, {
```

Comment:
I don't have experience with node-babel in particular, but ES6 is definitely coming and I'm using ES6/7 explicitly in my frontend projects. I'd +1 this :smile: